Digipost.Signature.Api.Client.Direct/DataTransferObjects/DataTransferObjectConverter.cs

```diff
@@ -65,12 +66,19 @@ public static JobStatusResponse FromDataTransferObject(directsignaturejobstatusr
 
         internal static directsignaturejobmanifest ToDataTransferObject(Manifest manifest)
         {
-            return new directsignaturejobmanifest
+            var dataTransferObject = new directsignaturejobmanifest
             {
                 sender = ToDataTransferObject(manifest.Sender),
                 document = ToDataTransferObject((Document) manifest.Document),
                 signer = ToDataTransferObject(manifest.Signer).ToArray()
             };
+            if (manifest.AuthenticationLevel != null)
```

Comment:
Litt flisespikkeri, men jeg er glad i newline før `if` og `return ...`, og ingen newline etter `return ...`. Men det viktigste er at det blir likt for denne og `directsigner` 🥇 .